lib/takeaway.rb

```diff
@@ -0,0 +1,79 @@
+require_relative './menu.rb'
+require_relative './order.rb'
+require_relative './twilio_client.rb'
+
+class Takeaway
+  attr_reader :current_order
+
+  def initialize(menu = Menu.new, current_order = Order.new, sms_client = TwilioClient.new)
+    @menu = menu
+    @current_order = current_order
+    @sms_client = sms_client
+  end
+
+  def see_menu
+    @menu.user_menu
+  end
+
+  def add_item(item, qty)
+    raise 'Please choose valid menu option' if not_on_menu?(item)
+    raise 'Quantity must be a valid integer' if not_a_number?(qty)
+
+    @current_order.add(item, qty)
+  end
+
+  def review_order
+    puts "Your order"
+
+    @current_order.basket.each do |item, qty|
+      subtotal = @menu.dishes[item] * qty
+      puts "x#{qty} #{item}, #{price_format(subtotal)}"
+    end
+
+    puts "Total: #{price_format(order_total)}"
+  end
+
+  def place_order(name, mobile)
+    process_order(name, mobile)
```

Comment:
What is the purpose of the `process_order` method? Since we don't call it anywhere except for on this line, we could reduce the number of methods we have and pull out the logic into `process_order` into `place_order` and delete `process_order` entirely